app/uk/gov/hmrc/trackingconsentfrontend/assets/package.json

```diff
@@ -8,7 +8,8 @@
   "scripts": {
     "start": "NODE_ENV=develop webpack-dev-server --config ./webpack.config.js --mode development",
     "build": "webpack --mode production",
-    "lint": "stylelint --fix \"**/*.scss\" && standardx --fix **/*.ts",
+    "lint": "stylelint \"**/*.scss\" && standardx **/*.ts",
+    "fixlint": "stylelint --fix \"**/*.scss\" && standardx --fix **/*.ts",
```

Comment:
Lint fix shouldn't change the semantics of the code, just style issues and formatting. I personally think it's fine to run as a pre-commit hook.

Reply:
So I'd argue strongly for not mutating code in any way during a commit.

The actual behaviour previously was that it mutated the code, the lint passed as a result but the changes made weren't included on the commit - therefore the commit itself had lint errors on it and I was left changes "not staged for commit".  If the goal is to keep the commit clean then running fix at the same time doesn't achieve that.

Reply:
We could use lint-staged https://medium.com/@okonetchnikov/make-linting-great-again-f3890e1ad6b8#.8qepn2b5l - this would also make the linting process faster. But I understand if you feel strongly about not mutating before committing.